### PR TITLE
Update README.md to better explain Syntax Theme installation

### PR DESCRIPTION
**Pull request by CodePurble**
I have added a clearer instruction to install the Syntax Theme usercss to avoid repeat issues like #224, #223, #221

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ Then:
 
 📦 [Install the usercss](https://github.com/StylishThemes/StackOverflow-Dark/raw/master/stackoverflow-dark.user.css). Supports automatic updates.
 
-## Additional styles
+## Additional styles (required for properly rendering blocks of code on StackOverflow pages)
```
Comment (the-j0k3r) [inline README.md:39]:
```suggestion
(required for properly rendering blocks of code on StackOverflow pages)
```

File: README.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ Then:
...
```
Comment (the-j0k3r) [inline README.md:38]:
```suggestion
## Additional styles
```

Comment (the-j0k3r) [inline README.md]:
```suggestion
## Additional styles
```

Not all additional styles will be required. This one is thats it.

**Comment by the-j0k3r** [comment]
There are a couple or required changes to make this future proof.

File: README.md (hunk 1 above)
Comment (CodePurble) [reply README.md]: Yes, this is okay with me.

File: README.md (hunk 2 above)
Comment (CodePurble) [reply README.md]: Yes, that is correct. Didn't look at it that way. :+1:

**Comment by CodePurble** [comment]
I have made the required edits. Please check.

File: README.md [hunk 3]
```diff
@@ -38,7 +38,8 @@ Then:
 ## Additional styles
 
 📦 [Install the StackOverflow Syntax Themes usercss](https://raw.githubusercontent.com/StylishThemes/Feature-Override-Styles/master/stackoverflow-syntax-themes.user.css)<br>
-This style allows you to choose preset syntax themes for StackOverflow and associated sites.<br><br>
+- This style allows you to choose preset syntax themes for StackOverflow and associated sites.
```
Comment (the-j0k3r) [inline README.md:41]:
```suggestion

```
I forgot about this sentence. Needs to be shortened the whole lot I think. Less reading better for all users apparently.

File: README.md [hunk 4]
```diff
@@ -38,7 +38,8 @@ Then:
...
+- The StackOverflow Syntax Themes style is **a required additional style** for properly rendering blocks of code on StackOverflow pages<br><br>
```
Comment (the-j0k3r) [inline README.md:42]:
```suggestion
- The StackOverflow Syntax Themes style is **a required additional style** for rendering blocks of code on StackOverflow pages it allows you to choose preset syntax themes.<br><br>
```

**Comment by the-j0k3r** [comment]
Better, though it could be further cleaned up. Will do for now.

**Comment by CodePurble** [comment]
Alright then

**Pull request merged**